python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -221,6 +222,8 @@ def getObservationMetaData(self, obsHistID=None, expDate=None, fieldRA=None, fie
 
                 nConstraints += 1
 
+        query += ' GROUP BY expMJD'
```

Comment:
Given the causality requirements of the AGN simulation, have you considered adding and ` ORDER BY expMJD` as well? Or does that slow doe the generation a lot?